modules/core/test/test_intrin_utils.hpp

```diff
@@ -890,7 +890,7 @@ template<typename R> struct TheTest
         }
         return *this;
     }
-
+        
```

Comment:
Git hook must be installed before development to avoid that.
Refer to "How to contribute" Wiki page.

Reply:
Removed.